testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewTest.m

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <XCTest/XCTest.h>
+#import "AppDelegate.h"
+
+@interface FlutterViewGLContextTest : XCTestCase
+@property(nonatomic, strong) FlutterViewController* flutterViewController;
+@property(nonatomic, strong) FlutterEngine* flutterEngine;
+@end
+
+#pragma clang diagnostic push
```

Comment:
can we move this to just around the offending area, so we don't open it up for the whole file?